src/main/java/io/confluent/copycat/jdbc/DataConverter.java

```diff
@@ -150,10 +148,10 @@ private static void addFieldSchema(ResultSetMetaData metadata, int col,
       // for single precision
       case Types.FLOAT:
```

Comment:
The JDBC float type is the same as double type. I think here we should combine them.